From 749b42f25534bec8625e74a6c1c61c3ad6cc8b17 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Tue, 16 Oct 2018 13:37:09 -0700
Subject: [PATCH 100/103] Add udf to t1700-probe-fs and to the manpage

If mkfs.udf is installed this will test to make sure that the filesystem
is detected.
---
 doc/C/parted.8          | 2 +-
 tests/t1700-probe-fs.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/C/parted.8 b/doc/C/parted.8
index eb7cd98..fecdc29 100644
--- a/doc/C/parted.8
+++ b/doc/C/parted.8
@@ -78,7 +78,7 @@ Create a new partition. \fIpart-type\fP may be specified only with msdos and
 dvh partition tables, it should be one of "primary", "logical", or "extended".
 \fIname\fP is required for GPT partition tables and \fIfs-type\fP is optional.
 \fIfs-type\fP can be one of "btrfs", "ext2", "ext3", "ext4", "fat16", "fat32",
-"hfs", "hfs+", "linux-swap", "ntfs", "reiserfs", or "xfs".
+"hfs", "hfs+", "linux-swap", "ntfs", "reiserfs", "udf", or "xfs".
 .TP
 .B name \fIpartition\fP \fIname\fP
 Set the name of \fIpartition\fP to \fIname\fP. This option works only on Mac,
diff --git a/tests/t1700-probe-fs.sh b/tests/t1700-probe-fs.sh
index b13cec0..853d6bc 100755
--- a/tests/t1700-probe-fs.sh
+++ b/tests/t1700-probe-fs.sh
@@ -23,7 +23,7 @@ dev=loop-file
 ss=$sector_size_
 n_sectors=$((512*1024))
 
-for type in ext2 ext3 ext4 btrfs xfs nilfs2 ntfs vfat hfsplus; do
+for type in ext2 ext3 ext4 btrfs xfs nilfs2 ntfs vfat hfsplus udf; do
 
   ( mkfs.$type 2>&1 | grep -i '^usage' ) > /dev/null \
       || { warn_ "$ME: no $type support"; continue; }
-- 
2.17.2

